src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -193,14 +191,17 @@ - (void)startNextRead {
   }
   __weak GRPCCall *weakSelf = self;
   __weak GRPCDelegateWrapper *weakWriteable = _responseWriteable;
-
+  
   dispatch_async(_callQueue, ^{
-    [weakSelf startReadWithHandler:^(grpc_event *event) {
-      if (!event->data.read) {
-        // No more responses from the server.
+    [weakSelf startReadWithHandler:^(NSDictionary *event) {
```

Comment:
I don't think many people would think of an "event" as a map. Let's pick a more descriptive name.